rust/bridge/shared/src/ffi/storage.rs

```diff
@@ -279,12 +282,12 @@ pub struct FfiSignedPreKeyStoreStruct {
 impl SignedPreKeyStore for &FfiSignedPreKeyStoreStruct {
     async fn get_signed_pre_key(
         &self,
-        prekey_id: u32,
+        prekey_id: DeviceId,
```

Comment:
Same thing, with a different set of identifiers for signed pre-keys.

Reply:
Fixed!